runelite-client/src/main/java/net/runelite/client/plugins/kourendlibrary/KourendLibraryPlugin.java

```diff
@@ -243,6 +244,10 @@ public void onChatMessage(ChatMessage event)
 				lastBookcaseAnimatedOn = null;
 			}
 		}
+		if (event.getType() == ChatMessageType.GAMEMESSAGE && Text.removeTags(event.getMessage()).startsWith("You hear the shifting of books"))
```

Comment:
Let's make the gamemessage type check a guard clause, since that's common between these if blocks. Also, getting the exact message would be handy here so we can just `.equals()` the strings.